src/thor/astar_bss.cc

```diff
@@ -84,7 +82,7 @@ void AStarBSSAlgorithm::Init(const midgard::PointLL& origll, const midgard::Poin
   // Reserve size for edge labels - do this here rather than in constructor so
   // to limit how much extra memory is used for persistent objects.
   // TODO - reserve based on estimate based on distance and route type.
-  edgelabels_.reserve(std::min(kInitialEdgeLabelCount, max_reserved_labels_count_));
+  edgelabels_.reserve(max_reserved_labels_count_);
```

Comment:
@kevinkreiser I checked in the PR that introduced this config option and I can't see any reasoning why it's not allowing to get past the hard-coded constants using `std::min`. I'd prefer to change it to respect the config option "blindly". WDYT?

Reply:
here the PR: https://github.com/valhalla/valhalla/pull/2719/files#diff-e7d2f8280a1f34d1175d125309288c693ea745ff14f88c3565378c77ed907bdd